e2e_tests/integrations/test_huggingface.py

```diff
@@ -161,3 +167,51 @@ def test_run_access_without_callback_configured(self):
 
         with pytest.raises(Exception):
             NeptuneCallback.get_run(trainer)
+
+    def test_log_parameters_indicator(self, environment):
+        with init_run(project=environment.project, api_token=environment.user_token) as run:
+            base_namespace = "custom/base/path"
+            callback = NeptuneCallback(run=run, base_namespace=base_namespace)
+            trainer = Trainer(**self._trainer_default_attributes, callbacks=[callback])
+            trainer.train()
+            run.sync()
+
+            assert run.exists(f"{base_namespace}/trainer_parameters")
+            assert run.exists(f"{base_namespace}/model_parameters")
```

Comment:
Don't we want to check if logged values are valid?

Reply:
Ok. Will try

Reply:
I've added a few simple one that we know they exists + In order to use a structure of `config` I've added proper model config that was introduced here: https://github.com/neptune-ai/neptune-client/pull/954